Essentials/src/com/earth2me/essentials/utils/NumberUtil.java

```diff
@@ -82,4 +82,8 @@ public static boolean isInt(final String sInt) {
         }
         return true;
     }
+    
+    public static boolean isInt(double decimal) {
```

Comment:
`isInteger` would likely be clearer - we're checking if it's a mathmatical integer, not the Java primitive type `int`.

Reply:
This isn't a proper check - usually we want to avoid hard 'equals' comparison using floating point math. I mentioned above why this fails at a UX level (42.0 is an int, but shouldn't be treated as one) but this also falls apart as the decimal gets extraordinarily large - eventually precision is lost and decimal - ((int)decimal) == 0 no longer holds true.